prov/verbs/src/fi_verbs.c

```diff
@@ -60,6 +60,12 @@ struct fi_provider fi_ibv_prov = {
 	.cleanup = fi_ibv_fini
 };
 
+struct util_prov fi_ibv_util_prov = {
+	.prov = &fi_ibv_prov,
+	.info = NULL,
+	.flags = UTIL_RX_SHARED_CTX,
```

Comment:
This change is unrelated, and as a result is hard to figure out its purpose

Reply:
I remove this form this commit

Reply:
I'd like to see the UTIL_RX_SHARED_CTX flag dropped in favor of using the domain attribute max_ep_srx_ctx value

Reply:
Will drop this flag